src/main/java/org/rumbledb/expressions/arithmetic/MultiplicativeExpression.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Authors: Stefan Irimescu, Can Berker Cikis
+ *
+ */
+
+package org.rumbledb.expressions.arithmetic;
+
+
+import org.rumbledb.exceptions.ExceptionMetadata;
+import org.rumbledb.exceptions.OurBadException;
+import org.rumbledb.expressions.AbstractNodeVisitor;
+import org.rumbledb.expressions.Expression;
+import org.rumbledb.expressions.Node;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class MultiplicativeExpression extends Expression {
+
+    public static enum MultiplicativeOperator {
+        MUL("*"),
+        DIV("div"),
+        MOD("mod"),
+        IDIV("idiv");
+
+        private String name;
+
+        MultiplicativeOperator(String name) {
+            this.name = name;
+        }
+
+        @Override
+        public String toString() {
+            return this.name;
+        }
+
+        public static MultiplicativeOperator fromSymbol(String symbol) {
+            if (symbol.contentEquals(MUL.toString())) {
+                return MUL;
+            }
+            if (symbol.contentEquals(DIV.toString())) {
+                return DIV;
+            }
+            if (symbol.contentEquals(MOD.toString())) {
+                return MOD;
+            }
+            if (symbol.contentEquals(IDIV.toString())) {
```

Comment:
Very interesting choice for the string comparison method. I was not aware such an alternative to the equals() method existed. I have looked into the comparison of the two, I would like to hear your motivation for using it in this context.

Reply:
This was unintentional (autocomplete). I have reverted.